### DISCUSSION TITLE
Transitive dependencies are not bumped in `Cargo.lock`

### DISCUSSION THREAD

**Discussion by torokati44** [renovatebot/renovate#23958] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

See for example this PR: https://github.com/ruffle-rs/ruffle/pull/12830
The bot bumped a few deps, and updated the lockfile accordingly.
However, right after merging it, locally running `cargo update` still made changes to `Cargo.lock`.

I wish the bot just ran `cargo update` before submitting the PR, to make sure the lockfile is entirely fresh. That's how we would like to use it at least.

Yes, I am aware of the `lockFileMaintenance` option, but it has it's own issues due to which we had to stop running it.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
Renovate is functioning as described and intended and there is no "problem" to report here.

Running `cargo update` on every PR would be undesirable for many so we will not do it by default, but you can create a new "Request a feature" discussion instead as this might be suitable for a postUpdateOption.
